bench-vortex/src/lib.rs

```diff
@@ -87,6 +87,26 @@ impl Format {
     }
 }
 
+#[derive(ValueEnum, Clone, Copy, Debug, Hash, Default, PartialEq, Eq)]
+pub enum Engine {
+    #[default]
+    Vortex,
```

Comment:
By engine I originally meant query engine. Not sure I agree with that Vortex should be part of that enum.

Reply:
How would you model results where no query engine exists?